src/main/resources/edu/cornell/kfs/paymentworks/businessobject/datadictionary/PaymentWorksVendor.xml

```diff
@@ -240,19 +238,25 @@
 	</bean>
 
 	<bean id="PaymentWorksVendor-requestingCompanyTelephone" parent="PaymentWorksVendor-requestingCompanyTelephone-parentBean" />
-	<bean id="PaymentWorksVendor-requestingCompanyTelephone-parentBean" abstract="true" parent="GenericAttributes-phoneNumberNoValidation">
+	<bean id="PaymentWorksVendor-requestingCompanyTelephone-parentBean" abstract="true" parent="AttributeDefinition">
 		<property name="name" value="requestingCompanyTelephone" />
 		<property name="label" value="Requesting Company Phone" />
 		<property name="required" value="true" />
-		<property name="maxLength" value="14" />
+		<property name="maxLength" value="25" />
```

Comment:
we talked, and Nancy is going double check the mac length in the DD versus what is defined in the database, erroring on the side of overly large fields as Payments doesn't always have food field size restrictions.

Reply:
Adjusted the DD to match actual table field sizes for these changes pertaining to DV vendors.